scripts/buildah-results.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+shopt -s inherit_errexit
+set -eu -o pipefail
+
+source "$(dirname ${BASH_SOURCE[0]})/common.sh"
+source "$(dirname ${BASH_SOURCE[0]})/buildah-common.sh"
+
+# function buildah_inspect() {
+#     buildah inspect --format='{{ .FromImageDigest }}' \
+#         ${1}
+# }
+
+# file storing image digest and tag
+filename="workspace/source/image_digest"
+
+if [ -e "$filename" ]; then
+  echo "File exists: $filename"
+else
+  echo "File does not exist: $filename"
+  exit 1
+fi
+
+phase "Extracting '${PARAMS_IMAGE}'  image digest"
+## Verify if the image is successfully built
+
+# Getting last 2 lines from filename
+tmp=$(tail -n 2 $filename)
+
+printf "%s" ${PARAMS_IMAGE} >${RESULTS_IMAGE_URL_PATH}
+printf "%s" ${tmp} >${RESULTS_IMAGE_METADATA_PATH}
+
+if [[ "${PARAMS_SKIP_PUSH}" == "false" ]]; then
+    printf "%s" $(cat workspace/source/image_digest_push) >${RESULTS_IMAGE_DIGEST_PATH}
```

Comment:
Any reason to to this in another step instead of directly writing in `/tekton/results/…` ? My guess might be a carriage return that buildah does, but it could be handle through a pipe in the script directly, couldn't it ?

Reply:
So in the `buildah push` command, when given the option `--digestfile`, it stores the image `SHA` value in it.  Here we are having a PARAM that gives us the choice to whether push the image to a registry or not. So I added the above line of code to add that image digest in results if we want to push it to the registry. Thus I'm handling the `bud` results and `push` results separately.

Reply:
> but it could be handle through a pipe in the script directly, couldn't it 
I'm not actually sure about this, like how to proceed with this

Reply:
ah make sense, let's keep it as is for now then 😉